module/VuFind/tests/integration-tests/src/VuFindTest/Mink/RecordActionsTest.php

```diff
@@ -946,6 +946,84 @@ function () use ($page, $checked) {
         $this->clickCss($page, '.logoutOptions a.logout');
     }
 
+    /**
+     * Test adding comments, rating and tag to a record and then
+     * delete them from user account menu
+     *
+     * @return void
+     */
+    public function testUserContentDeletion()
+    {
+        $this->changeConfigs(
+            [
+                'config' => [
+                    'Social' => [
+                        'rating' => true,
+                        'remove_rating' => true,
+                    ],
+                ],
+            ]
+        );
+
+        $page = $this->gotoRecord();
+        $this->waitForPageLoad($page);
+        $this->clickCss($page, '#loginOptions a');
+        $this->findCss($page, $this->openModalUsernameFieldSelector);
+        $this->makeAccount($page, 'username5');
```

Comment:
One more small issue: you need to clean up `username5` in `tearDownAfterClass` below, or else subsequent tests will fail.